ql/src/test/queries/clientpositive/cte_mat_1.q

```diff
@@ -1,10 +1,37 @@
 --! qt:dataset:src
 set hive.mapred.mode=nonstrict;
 set hive.optimize.cte.materialize.threshold=-1;
-set hive.explain.user=true;
 
 explain
 with q1(srcKey, srcValue) as (select * from src where key= '5')
+select a.srcKey, b.srcValue
+from q1 a join q1 b
+on a.srcKey=b.srcKey;
+
+set hive.optimize.cte.materialize.threshold=2;
+set hive.optimize.cte.materialize.full.aggregate.only=false;
+-- Use a format that retains column names
+set hive.default.fileformat=parquet;
```

Comment:
The default one positionally accesses physical columns, and we can't find some kinds of errors.